flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/streaming/connectors/kafka/table/KafkaDynamicTableFactoryTest.java

```diff
@@ -128,7 +132,7 @@
     static {
         KAFKA_SOURCE_PROPERTIES.setProperty("group.id", "dummy");
         KAFKA_SOURCE_PROPERTIES.setProperty("bootstrap.servers", "dummy");
-        KAFKA_SOURCE_PROPERTIES.setProperty("flink.partition-discovery.interval-millis", "1000");
+        KAFKA_SOURCE_PROPERTIES.setProperty("partition.discovery.interval.ms", "1000");
```

Comment:
Can we use `KafkaSourceOptions.PARTITION_DISCOVERY_INTERVAL_MS.key()` instead of a hard coded string?

Reply:
I assume the purpose of hard-coding here is that the test will fail if the option key is changed, which can alert that the option change might break user's DDL. I prefer to keep it for now.